### DISCUSSION TITLE
Folders for datasets?

### DISCUSSION THREAD

**Discussion by DanielThedie** [jeanollion/bacmman#7] [Ideas]
Hi! I've been enjoying using BACMMAN for a couple months now, and I see my list of Datasets steadily increasing. I thought it would be really nice to show them in "folders", i.e. have drop-down menus in the "Datasets" section. Is that a feature that would be possible to add? It would allow me to sort datasets by type of experiment and by date to navigate them more easily.

Cheers!

**Comment by jeanollion** [comment]
Hi Daniel, 
Thank you for your suggestion, that's a good idea. 
I would look for datasets in 2 or 3 sub-folder levels within the working directory and reflect the folder structure in the dataset section as you suggest. 
For the moment you can already sort your datasets in sub-folders and select those folders as working directories + when right-clicking on working directory you have the list of recent working directories which makes it a little bit faster to navigate the different sub-folders. 
Best, 
Jean

**Comment by jeanollion** [comment]
This feature has been added
